napari/layers/_image_layer/model.py

```diff
@@ -326,6 +332,32 @@ def interpolation_functions(self):
     def _clim_range_default(self):
         return [float(self.image.min()), float(self.image.max())]
 
+    def _update_thumbnail(self):
+        """Update thumbnail with current image data and colormap.
+        """
+        zoom_factor = np.divide(self._thumbnail_shape[:2],
+                                self._image_view.shape[:2]).min()
+        if self.multichannel:
+            zoomed = ndi.zoom(self._image_view, (zoom_factor, zoom_factor, 1),
+                              prefilter=False, order=0)
+            if self._image_view.shape[2] == 4:
+                mapped = zoomed
+            else:
+                ones = 255 * np.ones(list(zoomed.shape[:2]) + [1])
```

Comment:
```suggestion
                ones = 255 * np.ones(zoomed.shape[:2] + (1,))
```

Reply:
```suggestion
                ones = np.full(zoomed.shape[:2] + (1,), 255, dtype=np.uint8)
```

... raising the stakes in the suggestion game. =D

Reply:
oooh yes i missed that one!

Reply:
fixed